lib/kitchen/ancestor.rb

```diff
@@ -36,9 +36,15 @@ def increment_descendant_count(descendant_type)
     #
     # @param descendant_type [String, Symbol] the descendent's type
     # @param by [Integer] the amount by which to decrement
+    # @raise [RangeError] if descendant count is a negative number
```

Comment:
I'm not sure if RangeError was the right choice here. 
I was also considering Standard and Argument?

Reply:
that's right b/c that's what you're raising.